app/src/ui/app.tsx

```diff
@@ -583,6 +583,27 @@ export class App extends React.Component<IAppProps, IAppState> {
     this.props.dispatcher.setAppMenuState(menu => menu.withReset())
   }
 
+  private onTitlebarDoubleClick() {
+    if (__DARWIN__) {
```

Comment:
Let's rename this method to indicate it's Darwin-specific, and remove the redundant `__DARWIN__` check in here.